src/chat/ChatScreen.js

```diff
@@ -116,21 +116,18 @@ export default function ChatScreen(props: Props) {
   const showComposeBox = canSendToNarrow(narrow) && !showMessagePlaceholders;
 
   const titleBackgroundColor = useSelector(state => getTitleBackgroundColor(state, narrow));
-  const orientation = useSelector(state => getSession(state).orientation);
   const insets = useSafeAreaInsets();
 
   return (
     <ActionSheetProvider>
       <View style={[componentStyles.screen, { backgroundColor }]}>
         <KeyboardAvoider style={styles.flexed} behavior="padding">
-          {orientation === 'PORTRAIT' && (
```

Comment:
I can't think of a device that has a nonzero top inset in landscape
    mode, so it's possible that this commit is NFC. Still, we might as
    well (1) handle that case, so we don't have to think about it, and
    (2) remove something that can cause confusion.

Hmm, maybe a foldable? Or some odd-shaped tablet?

In any case I definitely agree with this reasoning.